packages/shared/src/components/Feed.tsx

```diff
@@ -123,11 +123,12 @@ export default function Feed<T>({
   const insaneMode = !forceCardMode && listMode;
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
   const isSquadFeed = feedName === 'squad';
+  const { shouldUseFeedLayoutV1 } = useFeedLayout({ feedName });
   const { items, updatePost, removePost, fetchPage, canFetchMore, emptyFeed } =
     useFeed(
       feedQueryKey,
       currentSettings.pageSize,
-      isSquadFeed ? 3 : currentSettings.adSpot,
+      isSquadFeed || shouldUseFeedLayoutV1 ? 3 : currentSettings.adSpot,
```

Comment:
```suggestion
      isSquadFeed || shouldUseFeedLayoutV1 ? 2 : currentSettings.adSpot,
```

This should also be adjusted to use the 3rd spot - which in this case would be nr 2. 

Nr.2 is the default in the settings so just check to align these 🙏

Reply:
Thanks for the suggestion! 🚀